README.md

```diff
@@ -97,17 +97,18 @@ Duplicate `user` entries are not accepted, and only the first instance of that `
 -----
 
 ### edit_backends.py
-**What it does**: Adds or Removes a backend from a project within your Hub</br>
+**What it does**: Adds or Removes a backend from a Group or Project within your Hub</br>
 **Parameters**:</br>
 &ensp;`<hub>`: Required. The name of your hub.</br>
 &ensp;`<action>`: Required. Either `add` or `remove`.</br>
 &ensp;`<group>`: Required. The name of the parent group.</br>
-&ensp;`<project>`: Required. The name of the parent project.</br>
 &ensp;`<backend>`: Required. The name of the backend to add or remove.</br>
+&ensp;`<-project>`: Optional. The name of the parent project.</br>
 &ensp;`<-priority>`: Required for `add` action. Integer priority of the backend, between 1 and 10,000.</br>
 
 **Usage**:</br>
-&ensp;`python edit_backends.py <hub> <action> <group> <project> <backend_name> <-priority>`
+&ensp;If adding: `python edit_backends.py <hub> add <group> <backend_name> -project <project> -priority <priority>`
```

Comment:
A `</br>` tag should be added at the end of this line for the line break to appear between the two usage options.